KBH/10Day/다트게임.js

```diff
@@ -0,0 +1,52 @@
+function solution(dartResult) {
+
+    const rounds = dartResult.match(/\d+[SDT][*#]?/g);
+
+    let scores = [];
```

Comment:
참조 변수의 경우 우선 `const`로 선언해주는것이 좋습니다!
```suggestion
    const scores = [];
```